lib/services/plugins-service.ts

```diff
@@ -178,32 +216,33 @@ export class PluginsService implements IPluginsService {
 	private createPluginsData(pluginsService: ICordovaPluginsService): IFuture<void> {
 		return (() => {
 			var plugins = pluginsService.getAvailablePlugins().wait();
-
 			_.each(plugins, (plugin: Server.CordovaPluginData) => {
-
 				try {
-					var pluginData = pluginsService.createPluginData(plugin).wait();
-				} catch(e) {
+					var data = pluginsService.createPluginData(plugin);
```

Comment:
This is considered bad coding practice and is explicitly mentioned in our coding guidelines: https://github.com/Icenium/Ice/wiki/JavaScript-Coding-Guidelines#all-variables-should-be-declared-on-the-top-of-the-function-with-only-one-var